packages/core/src/App/initStore.js

```diff
@@ -19,6 +19,9 @@ const setStorageEvents = root_store => {
                 break;
             }
             case 'active_loginid':
+                if (localStorage.getItem('active_loginid') === 'null') {
```

Comment:
```suggestion
                if (localStorage.getItem('active_loginid') === null) {
```

@mahdiyeh-fs Shouldn’t be `null`? 🤔 

<img width="393" alt="image" src="https://user-images.githubusercontent.com/72082844/182794254-ee244a9b-7ccd-46e6-bee0-dfbb743bcce4.png">

Reply:
@farzin-fs localStorage keys and values are always strings 🙂
@mahdiyeh-fs just to confirm, is this what you can get here: a 'null' string? If yes, I can suggest an equivalent expression:
```suggestion
                if (!JSON.parse(localStorage.getItem('active_loginid'))) {
```

Reply:
@farzin-fs As @maryia-binary said we have strings in localStorage.
@maryia-binary yes, that would be better, applied.

Reply:
@maryia-binary Yes, keys and values are always string, But that doesn’t mean if you ask for a key that doesn’t exist you will still get a string.

```js
/** Returns the current value associated with the given key, or null if the given key does not exist. */
getItem(key: string): string | null;
```

---

As checked with @mahdiyeh-fs we found that we are intentionally setting `active_loginid` to the string of `null` in the `cleanUp` action of `packages/core/src/Stores/client-store.js` instead of removing the key from local storage, So the code above is valid for this case 🙇🏻